frontend/src/views/cluster/Overview/components/OverviewContent.vue

```diff
@@ -173,6 +174,15 @@ const machineLockedForKubernetesUpgrade = computed(() => {
 onMounted(async () => {
   isEmbeddedDiscoveryServiceAvailable.value = await embeddedDiscoveryServiceFeatureAvailable()
 })
+
+const isMinorDowngrade = (
+  fromVersion: string | undefined,
+  toVersion: string | undefined,
+): boolean => {
+  if (fromVersion == undefined || toVersion == undefined) return true
```

Comment:
always always <ins>___always___</ins> triple equals in javascript `===`

that said, you can just do this here
```suggestion
  if (!fromVersion || !toVersion) return true
```